src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/methodparampad/InputWhitespaceAround.java

```diff
@@ -0,0 +1,31 @@
+package com.puppycrawl.tools.checkstyle.checks.whitespace.methodparampad;
+
+@SuppressWarnings({"this", "that"})
+public class InputWhitespaceAround
```

Comment:
Class and file name are still incorrect.
Input should be named after test. Test is named `MethodParamPadCheckTest`, Input should be named `InputMethodParamPad...`.